### ISSUE TITLE
Test server not launching on tests

### ISSUE THREAD

**Issue by gregtap** [weluse/django-nose-selenium#6]
Hello,

When I do python manage.py test, I can see firefox openning two windows but the second one gives me an Unable to connect on 127.0.0.1:8000

As if the test did not lunch the manage.py runserver by itself.

If I launch manage.py runserve on an other terminal and run my tests it works.

Is that the normal behavior ?

**Comment by passy** [comment]
Did you run the test with either the `--with-cherrypyliveserver` or `--with-djangoliveserver` flag?

**Comment by pelletier** [comment]
Yes, with `--with-djangoliveserver` (we work together). We are trying hard to set up Selenium tests for our Django application but no success for now :(

**Comment by passy** [comment]
That is odd. Did you set the `LIVE_SERVER_PORT` to 8000 in the config?

**Comment by ogirardot** [comment]
it did not work for me either until i set the LIVE_SERVER_PORT = 8000 in my settings
